sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -24,35 +24,23 @@
         <Subsystem>CBP</Subsystem>
         <Version>3.8</Version>
         <Author>ecoughlin</Author>
-        <EFDB_Topic>CBP_command_moveAzimuth</EFDB_Topic>
-        <Alias>moveAzimuth</Alias>
+        <EFDB_Topic>CBP_command_move</EFDB_Topic>
+        <Alias>move</Alias>
         <Device></Device>
         <Property></Property>
         <Action></Action>
         <Value></Value>
         <Explanation>http://sal.lsst.org</Explanation>
```

Comment:
Please replace all Explanation tags with Description tags.

In this case something like "Move the CBP to the specified elevation and azimuth" (or swap those axes).